de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/InitializationCallbacks.java

```diff
@@ -39,7 +41,7 @@
         private final DesktopPresenterImpl presenter;
         private final UserSessionServiceFacade userSessionService;
         private final UserPreferencesCallback userPreferencesCallback;
-        Logger LOG = Logger.getLogger(BootstrapCallback.class.getName());
+        Logger LOG = Logger.getLogger("BootstrapCallback");
```

Comment:
I thought we were intentionally getting the class name here, in case the name is refactored, then the logger will use the new class name?